packages/page-staking/src/Nomination/StahesTable.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Table } from '@polkadot/react-components';
+import NominatedAccount from '@polkadot/app-staking/Nomination/NominatedAccount';
+import { useApi, useCall } from '@polkadot/react-hooks';
+import { useTranslation } from '@polkadot/app-staking/translate';
+import { ActiveEraInfo, EraIndex } from '@polkadot/types/interfaces';
+import { Option } from '@polkadot/types';
+import { DeriveStakingOverview, DeriveStakerReward } from '@polkadot/api-derive/types';
+
+interface Props {
+  allRewards?: Record<string, DeriveStakerReward[]>;
+  controllerAccountId?: string | null;
+  className?: string;
+  allStashes?: string[];
+  isVisible: boolean;
+  next?: string[];
+  stakingOverview?: DeriveStakingOverview;
+  ownStashes?: [string, boolean][];
+  onUpdateControllerState: (controllerAlreadyBonded: boolean) => void;
+  onUpdateNominatedState: (controllerAlreadyBonded: boolean) => void;
+}
+
+function StashesTable({ className, allStashes, isVisible, next, allRewards, stakingOverview, controllerAccountId, ownStashes, onUpdateControllerState, onUpdateNominatedState }: Props): React.ReactElement<Props> {
```

Comment:
Duplicated code again. Single place, single operation, single long-term maintenance. (This is now already outdated as it stands). We cannot have this in here.